generator/generator_tool/generator_tool.cpp

```diff
@@ -80,6 +85,14 @@ DEFINE_bool(disable_cross_mwm_progress, false,
 DEFINE_string(srtm_path, "",
               "Path to srtm directory. If set, generates a section with altitude information "
               "about roads.");
+DEFINE_string(countries_dir, "",
+              "Directory with countries file which describes mwm tree. Used for routing and cross mwm "
```

Comment:
I thought there's no need in this flag, please, use already existing data_path flag. Countries.txt should be inside data_path.